### ISSUE TITLE
[Suggestion] Making shields less intrusive

### ISSUE THREAD

**Issue by capybar0** [tr7zw/FirstPersonModel#509]
To avoid shield clipping issues (like the screenshot bellow) I think it would be interesting if we had an option to **change the X position of the camera while holding up the shield (holding right click)**. This way, we could have a less intrusive shield without having to change the default X axis from our player standing or sneaking, which I don't like 'cause it makes me see see my neck when I look down.
![intrusiveshield](https://github.com/user-attachments/assets/98a88b07-38a2-4feb-a3fa-3d59751696b8)

**Comment by JumboGeronimo** [comment]
Yeah some more options for the camera or other changes to make items like the Shield less obtrusive when looking up

![Image](https://github.com/user-attachments/assets/2a5629ad-2b58-4e3e-8e25-80773b3f5173)

**Comment by tr7zw** [comment]
There won't be camera options as the mod doesn't touch the camera. And removing that the shield goes up when blocking up also doesn't make much sense. So from my point of view: works as intended. This also has been the tagline in the mod description for the past 5 years 😅 

![Image](https://github.com/user-attachments/assets/e6bdab20-962c-45b8-a21d-ea4e54630915)

**Comment by capybar0** [comment]
@tr7zw but my suggestions is just about changing the camera X axis WHEN blocking with a shield. The option to change the camera axis for standing or sneaking **is already something present in the mod.**

**Comment by tr7zw** [comment]
No it's not. The offset sliders change the Model positon relative to the vanilla camera. So changing that offset the camera then sees the top of the torso/might get clipped into armor or other equipment of the player. You can play around with the setting to simulate how that would look like, but my best guess is that you can see your own back at the point that the shield is not too big anymore.

**Comment by capybar0** [comment]
@tr7zw makes sense, I had to make ajustements for the camera to don't clip in inside the armor. Probably with the option I suggested, we would have a less intrusive shield, but with the consequence of having the camera clip inside the armor, so at the end it doesn't really help much. Thanks for the reply!

**Comment by tr7zw** [comment]
I think a decent solution is to use the latest vanilla changes (or mods) to replace the shield model with a less giant one. There should be resourcepacks out there.
